components/sub-components/Articles.jsx

```diff
@@ -0,0 +1,73 @@
+import { IoChatbubbleOutline } from "react-icons/io5";
+import { FaRegHeart } from "react-icons/fa";
+import styles from "./Articles.module.css";
+import { lineSpinner } from "ldrs";
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../../utils/api.js";
+import { Link } from "react-router-dom";
+
+lineSpinner.register();
+
+const Articles = ({setVotes, setLoading, loading}) => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    setLoading(true);
+    fetchArticles().then((fetchedArticles) => {
+      setLoading(false);
+      setArticles(fetchedArticles);
+    });
+  }, []);
+
+  return loading ? (
+    <div className={styles.loading_symbol}>
+      <l-line-spinner
+        size="40"
+        stroke="3"
+        speed="1"
+        color="black"
+      ></l-line-spinner>
+    </div>
+  ) : (
+    <ol className={styles.body}>
+      {articles.map((article) => {
+        const {
+          article_id,
+          title,
+          topic,
+          author,
+          votes,
+          created_at,
+          comment_count,
+          article_img_url,
+        } = article;
+        const date = new Date(created_at)
+        return (
+          <li id={article_id} key={article_id}>
+              <Link to={`/${article_id}`} onClick={()=>{setVotes(votes)}}>
+              <div className={styles.article_card}>
+                <div className={styles.author}>{author}</div>
+                <div className={styles.created_at}>{date.toLocaleDateString()}</div>
+                <div className={styles.text}>
+                  <div className={styles.title}>
+                    <h4>
+                      <b>{title}</b>
+                    </h4>
+                  </div>
+                <div className={styles.topic}>
+                  <i># {topic}</i>
+                </div>
+                </div>
+                <img className={styles.img} src={article_img_url} alt={title} />
+                <div className={styles.comment_count}><IoChatbubbleOutline />{comment_count}</div>
+                <div className={styles.votes}><FaRegHeart />{+votes}</div>
+              </div>
+            </Link>
```

Comment:
This is a lot of loose HTML code in your component, some of this could be bundled into a React component, such as `ArticleCard` for each `<li>` to be displayed in the list